docs/app/Examples/modules/Dropdown/Content/DropdownExampleDescription.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Dropdown } from 'semantic-ui-react'
+
+const DropdownExampleDescription = () => (
+  <Dropdown text='Filter' floating labeled button className='icon'>
+    {/* <i class="filter icon"></i> */}
```

Comment:
The icons are commented and I'm manually adding the classname `icon` because we don't currently support this. This is true for **all** labeled button examples, of which there are many in this documentation. I could add a message/comment to all of them I guess but that seems a little heavy. Thoughts on how to address?